### ISSUE TITLE
Input not in a form field not a failure of WCAG??

### ISSUE THREAD

**Issue by DavidMacDonald** [wet-boew/wet-boew-wpss#70]
A recent crawl gave 3200 errors for a search field not inside a form.. it seems to work fine. Why is it a WCAG failure?

http://paralympique.ca/%C3%A0-propos-de-sotchi 

    Line: 295 Column:  24
    <input> found outside of a <form>
    Source line: <input type='radio' name='disptarget' id='dm_mobile' />

**Comment by irwink** [comment]
This is similar to issue #69 F68 fails select boxes for not being wrapped in a form, in that a 'form' control is outside a form.  My thinking was, when I coded the check, that there is no reason to have a control outside of a form and that this may lead to confusion for some users.  Since there are some cases when it does work (e.g. JavaScript providing the actions), and in order to reduce the possibility of false errors, I will update the tool not not report controls outside of a form as an error.

The change will appear in the next version of the tool.

**Comment by irwink** [comment]
Addressed in version 5.1.0.
